src/themes/website/styles/gql.scss

```diff
@@ -0,0 +1,318 @@
+        height: 670px;
+        &.with-ws {
+            height: 295px;
+        }
+        @media (max-width: 1200px) and (min-width: 769px) {
+            height: 218px;
+            &.with-ws {
+                height: 218px;
+            }
+        }
+        @media (max-width: 768px) {
+            height: 600px;
+            &.with-ws {
+                height: 280px;
+            }
+        }
+    }
+    @media (min-height: 1201px) {
+        height: 860px;
+        &.with-ws {
+            height: 392px;
+        }
+        @media (max-width: 1200px) and (min-width: 769px) {
+            height: 254px;
+            &.with-ws {
+                height: 254px;
+            }
+        }
+    }
+}
+
+.response-ws {
+    @media (max-height: 1080px) {
+        height: 231px;
+        &.with-spinner {
+            height: 207px;
+        }
+    }
+    @media (max-height: 1200px) and (min-height: 1081px) {
+        height: 311px;
+        &.with-spinner {
+            height: 287px;
+        }
+        @media (max-width: 1200px) and (min-width: 769px) {
+            height: 218px;
+        }
+        @media (max-width: 768px) {
+            height: 280px;
+        }
+    }
+    @media (min-height: 1201px) {
+        height:  404px;
+        &.with-spinner {
+            height: 380px;
+        }
+        @media (max-width: 1200px) and (min-width: 769px) {
+            height: 254px;
+        }
+    }
+}
+
+.explorer-operations {
+    @media (max-height: 1080px) {
+        height: 464px;
+        @media (max-width: 1200px) and (min-width: 769px) {
+            height: 1044px;
+        }
+        @media (max-width: 768px) {
+            height: 400px;
+        }
+    }
+    @media (max-height: 1200px) and (min-height: 1081px) {
+        height: 614px;
+        @media (max-width: 768px) {
+            height: 400px;
+        }
+    }
+    @media (min-height: 1201px) {
+        height: 804px;
+        @media (max-width: 768px) {
+            height: 400px;
+        }
+    }
+}
+
+.gql-console-height {
+    height: 100%;
+}
+
+.gql-list-group {
+    list-style: none;
+    padding-left: 10px;
+
+    li {
+        border-top: none;
+    }
+}
+
+.mw-600 {
+    flex: 0 0 100%;
+    max-width: 100%;
+    @media (min-width: 901px) {
+        flex: 0 0 600px;
+        max-width: 600px;
+    }
+}
+
+.margin-top-variables {
+    margin-top: 30px !important;
+}
+
+.margin-top-send {
+    margin-top: 15px;
+}
+
+.breadcrumb-gql {
+    margin-right: 20px;
+}
+
+.input-node-color {
+    color: rgb(139, 43, 185);
+}
+
+.not-input-node-color {
+    color: #0F7EBD;
+}
+
+.icon {
+    padding: 10px;
+    color: #0078D4;
+    min-width: 50px;
+    text-align: center;
+}
+
+.presentation-item {
+    padding: 2px 10px;
+
+    label {
+        margin-bottom: 0;
+    }
+
+    select {
+        background-color: white;
+        color: rgb(15, 126, 189);
+    }
+}
+
+.headers-gql {
+    height: 60px;
+}
+
+.operationType-gql {
+    width: 100%;
+    color: rgb(177, 26, 4);
```

Comment:
Please make a variable out of `rgb(177, 26, 4);` and all other colors, and add it to variables.scss. This reduces duplicates and will help with ongoing deep customizations work.